src/proto/FunctionRpc.proto

```diff
@@ -298,6 +298,9 @@ message RpcFunctionMetadata {
 
   // A flag indicating if managed dependency is enabled or not
   bool managed_dependency_enabled = 14;
+
+  // Properties for function metadata
```

Comment:
Can we add a bit more context to the comment? For example, if I understand the feature correctly this is perhaps what I would put:

```suggestion
  // Properties for function metadata
  // They're usually specific to a worker and largely passed along to the controller API for use outside the host (e.g. the portal may adjust its experience based on these properties)
```